poleno/utils/tests/test_http.py

```diff
@@ -64,14 +63,12 @@ def test_regular_file(self):
         self._check_response(response, FileResponse, 200)
         self._check_content(response, path)
 
-    @unittest.skip(u'FIXME')
     def test_directory_raises_exception(self):
-        with self.assertRaisesMessage(OSError, u'Not a regular file: /'):
+        with self.assertRaises(TypeError):
```

Comment:
Po tom co subor nenajde nam vrati objekt `([<RegexURLPattern None ^file/$>])`, typu `tuple(list)` a nasledne vyhodi chybu `unhashable type list`. Skusil som to obijst a natvrdo pretypovat objekt na tuple, vtedy som sa dostal o krok dalej a hodilo chybu `NoReverseMatch`.

Reply:
Nad tymto sa budem musiet este zamysliet. Samotne `send_file_response` pri neexistujucom subore raisne exception `OSError`, ale potom Django test client nejak nezvladne ten exception spracovat.

Bude sa treba pozriet, co vlastne spravia endpointy pouzivajuce `send_file_response`, ked `send_file_response` raisne exception. Ci sa nejak sialene zrubu, ale slusne vratia 500 Internal error alebo nieco podobne.

A pripadne upravit samotne `send_file_response`, resp. viewy, ktore ho pouzivaju, aby namiesto raisovania exceptions vracali 500 Internal error, alebo 404 Not found.

Reply:
@viliambalaz Zistene v com je problem. Ked endpoint raisne exception, tak django do response vyrendruje template `500.html`. Vid [source](https://github.com/django/django/blob/1.7.4/django/views/defaults.py#L36-L47).

Ked rucne premazes obsah `chcemvediet/templates/500.html` a potom pustis testy, tak test zbehne. Cize pre to, aby nam testy fungovali potrebujeme bud namockovat prazdny obsah pre template `500.html`, alebo dosiahnut, aby `loader.get_template('500.html')` ten template nenasiel.

Reply:
:ok: prepisal som `settings.TEMPLATE_LOADERS`.

Reply:
@viliambalaz  Opat `poleno.utils.template.TranslationLoader` je tu zbytocny.

Reply:
:ok: